src/MapImportedContent.php

```diff
@@ -170,16 +185,67 @@ protected function castAttribute(string $column, string $attribute, string $mode
             }
 
             if (is_array($callback)) {
-                return (new ValidationPipeline)($value, $callback);
+                return (new CastingPipeline)($value, $callback);
             }
         }
 
         return $value;
     }
 
+    protected function validateAttribute(string $column, string $attribute, string $model, array $row): ?bool
+    {
+        $value = array_key_exists($column, $row) ? $row[$column] : null;
+
+        if (!$this->validators) {
+            return true;
+        }
+
+        $validation = $this->validators->filter(function ($value, $key) use ($model) {
+            return $key === $model;
+        });
+
+        if (!$validation) {
+            return true;
+        }
+
+        $modelValidations = Arr::get($validation, $model, []);
+
+        if (array_key_exists($attribute, $modelValidations)) {
```

Comment:
Better to wrap the validation alone and casting alone

Reply:
done @simoebenhida